src/parse.js

```diff
@@ -4,7 +4,7 @@ import ColorSpace from "./space.js";
 
 // CSS color to Color object
 export default function parse (str) {
-	let env = {str};
+	let env = {"str": str?.trim()};
```

Comment:
Actually, we may want to coerce to a string here as well (`str + ""`) before trimming.

Reply:
Something like this?
```diff
-let env = {"str": str?.trim()};
+let env = {"str": (str+"")?.trim()};
```
The parentheses could also be removed (just preference).